src/popen.cpp

```diff
@@ -79,7 +79,7 @@ FILE * popen(const char *command, const char *mode)
   }
 
   {
-    WRAPPER_EXECUTION_DISABLE_CKPT();
+    WrapperLock wrapperLock;
```

Comment:
Thanks for the extra comment.  As I understand it, DISABLE/ENABLE_CKPT() is replaced by a local variable, 'wrapperLock', with a constructor and an implicit destructor when the local variable goes out of scope.

Can we think of a more meaningful variable name, or else a standard comment to always attach to this usage?  In particular, we've lost the word 'CKPT'.  So, a casual reader will understand only that we're acquiring a lock, and will not understand that we're disabling checkpoints.  Here is one possible pattern, but I'm open to other patterns that will be meaningful to those not used to C++.

```
// C++: local variable implicitly calls lock constructor, and later implicitly calls destructor when the variable is out of scope.
WrapperLock lockToDisableCheckpoints;
```

I'm absolutely sure that many people who aren't used to the niceties of C++ (e.g., new Ph.D. students) will otherwise be totally confused by this implicit use of a constructor/destructor.

Reply:
Renamed to `disableCheckpoint` since "lock" is implicit in the type.